src/homekit.cs

```diff
@@ -75,6 +79,14 @@ partial interface HMHomeManagerDelegate {
 
 		[Export ("homeManager:didRemoveHome:"), EventArgs ("HMHomeManager")]
 		void DidRemoveHome (HMHomeManager manager, HMHome home);
+
+		[iOS (13,0), NoWatch, NoTV, NoMac]
+		[Export("homeManager:didReceiveAddAccessoryRequest:")]
```

Comment:
```suggestion
		[Export("homeManager:didReceiveAddAccessoryRequest:"), EventArgs ("HMHomeManager")]
```

Reply:
Can I have some explanation about this one ? Or maybe a link to the documentation ? I had no idea about this one 👀

Reply:
@joconte https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#generating-events 😉